tests/js/spec/views/projectDebugFiles.spec.jsx

```diff
@@ -1,54 +0,0 @@
-import React from 'react';
-
-import {shallow, mount} from 'sentry-test/enzyme';
-import {Client} from 'app/api';
-import ProjectDebugFiles from 'app/views/settings/projectDebugFiles';
-
-const ENDPOINT = '/projects/org/project/files/dsyms/';
-
-describe('ProjectDebugFiles', function() {
-  beforeEach(function() {
-    Client.clearMockResponses();
-    Client.addMockResponse({
-      url: '/projects/org/project/',
-      body: {},
-    });
-  });
-
-  it('renders empty', function() {
-    Client.addMockResponse({
-      url: ENDPOINT,
-      body: [],
-    });
-    const wrapper = shallow(
-      <ProjectDebugFiles
-        params={{orgId: 'org', projectId: 'project'}}
-        location={{query: {}}}
-      />,
-      TestStubs.routerContext()
-    );
-
-    expect(wrapper).toMatchSnapshot();
-  });
-
-  it('renders', function() {
-    Client.addMockResponse({
-      url: ENDPOINT,
-      body: TestStubs.DebugSymbols(),
-    });
-    Client.addMockResponse({
-      url: ENDPOINT,
-      method: 'PUT',
-    });
-
-    const wrapper = mount(
-      <ProjectDebugFiles
-        params={{orgId: 'org', projectId: 'project'}}
-        location={{query: {}}}
-      />,
-      TestStubs.routerContext()
-    );
-
-    expect(wrapper).toMatchSnapshot();
```

Comment:
I think it's worth having tests, just without `toMatchSnapshot()` just to test that it renders properly without errors/warnings.

Reply:
Yeah, I agree - I removed it because it was causing me some weird issues and we are covered on this page with Percy anyway. Today I rewrote the test from scratch to cover basic rendering 👍